lib/internal/process.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 const errors = require('internal/errors');
+const util = require('util');
```

Comment:
I've checked this. If I don't miss something, this module is [required only in the `bootstrap_node.js`](https://github.com/nodejs/node/blob/0d22858d67f5f8f7959a55ceca23adafe12827d5/lib/internal/bootstrap_node.js#L36) and the function which requires `util` [is called there unconditionally](https://github.com/nodejs/node/blob/0d22858d67f5f8f7959a55ceca23adafe12827d5/lib/internal/bootstrap_node.js#L55).